app/services/event_driven_services.py

```
@@ -0,0 +1,766 @@
+                if cpu_percent > 85 or memory.percent > 85:
+                    await self.publish_event(
+                        EventType.SYSTEM_HEALTH,
+                        {**health_data, "alert": "high_resource_usage"}
+                    )
+                
+                logger.debug("Health monitoring complete", **health_data)
+                
+            except Exception as e:
+                logger.warning("Health monitoring failed", error=str(e))
+    
+    async def _handle_cleanup(self, data: Optional[Dict], source: str, message_id: Optional[str] = None):
```

Ruff (0.12.2)

Unused method argument: `data`

(ARG002)